assets/scripts/lobby/sockets/sockets4.js

```diff
@@ -176,26 +176,28 @@ socket.on("openModModal", function (data) {
 });
 
 var currentOnlinePlayers;
-socket.on("update-current-players-list", function (currentPlayers) {
+socket.on("update-current-players-list", function (currentPlayersAndStatuses) {
     // console.log("update the current player list request received");
     // console.log(currentPlayers);
     //remove all the li's inside the table
     $("#current-players-table tbody tr td").remove();
     $("#current-players-table tbody tr").remove();
 
     // currentOnlinePlayers = currentPlayers;
-    autoCompleteStrs = currentPlayers;
+    autoCompleteStrs = currentPlayersAndStatuses.map(cp => cp[0]);
```

Comment:
Don't fully understand this line yet. On another note, please use the function(var){} format as opposed to the var => var format. I'm ok with the short hand if else notation you use, but I'd like the full function(var){} format please :).

Reply:
Actually I think `=>` will be ok here. I've found what the Array.prototype.map function does and it would look unnecessarily long with the full function(){} format.